index.html

```diff
@@ -6,8 +6,67 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Document</title>
     <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/7cbe454c49.js" crossorigin="anonymous"></script>
 </head>
 <body>
+    <header class="header-container">
+    <div class="sub-container">
+        <a href="#" class="logo-cont">Olamarx</a>
+      <button type="submit" class="header-font"><i class="fa fas-solid fa-bars font"></i> </button> 
+    </div>
+    </header>
+
+    <section>
+        <div class="main-container">
+            <div class="images">
+
+                <div class="img left-img">
+                    <img src="./images/Rectangle6.png" alt="Rectangle">
+                    <img src="./images/Ellipse11.png" alt="Ellipse">
+                </div>
+
+                <div class="img center-img">
+                    <img src="./images/Rectangle7.png" alt="Rectangle">
+                    <img src="./images/Ellipse5.png" alt="Ellipse">
+                </div>
+
+                <div class="img rigth-img">
+                    <img src="./images/Rectangle4.png" alt="Rectangle">
+                    <img src="./images/Ellipse3.png" alt="Ellipse">
+                </div>
+
+            </div>
+        </div>
+<div class="porfolio-content">
+    <div class="portfolio-h1">
+        <h1>I'm Olamarx </h1>
+        <h1 class="span-portfolio-h1-1">Glad to have</h1>
+        <h1 class="span-portfolio-h1-2">you here!</h1> 
+    </div>
+<div class="portfolio-content-para">
+    <p class="portfolio-para">
+        I am a Web Developer! I can help you build a standard and responsive website. 
+        You can look through some of my handiworks and experience. <span class="portfolio-para-span">Do you like what you see?</span>
+   LET US CONNECT </p>
+</div>
+</div>
+
+
+<p class="connect-with-me">
+        CONNECT WITH ME
```

Comment:
- [ ] Please make sure to stick to the original [figma](https://github.com/Olamarx/Microverse-Portfolio-setup/pull/1/files) design guidelines(fonts, margins, paddings, colors etc)
- [ ] Your design should look identical to the original design. At the moment, it does not. Please make adjustments to match the design

### Your design
![image](https://user-images.githubusercontent.com/43995830/158801651-8ba21ed3-029e-4d46-ab6b-55ed2f1d7a03.png)

### Original Design
![image](https://user-images.githubusercontent.com/43995830/158801739-47ff0a42-2bb0-4cb5-940e-ff311bcd104f.png)